biosimulations/apps/dispatch-service/src/app/services/sbatch/sbatch.service.ts

```diff
@@ -25,8 +25,8 @@ export MODULEPATH=/isg/shared/modulefiles:/tgcapps/modulefiles
 source /usr/share/Modules/init/bash
 module load singularity/3.1.1
 date
-\`wget ${apiDomain}run/${simId}/download -O ${tempSimDir}/in/${omexName} 1>${tempSimDir}/out/job.output 2>&1\`
-command=\\"singularity run -B ${tempSimDir}/in:/root/in -B ${tempSimDir}/out:/root/out /home/FCAM/crbmapi/nfs/biosimulations/singularity_images/${simulator} -i /root/in/${omexName} -o /root/out\\"
+\`wget ${apiDomain}run/${simId}/download -O "${tempSimDir}/in/${omexName}" 1>${tempSimDir}/out/job.output 2>&1\`
+command=\\"singularity run -B ${tempSimDir}/in:/root/in -B ${tempSimDir}/out:/root/out /home/FCAM/crbmapi/nfs/biosimulations/singularity_images/${simulator} -i "/root/in/${omexName}" -o /root/out\\"
```

Comment:
Could you revert this change? 
I can adress #1901 at the same time as #1891 since this service will be changing. Currently this conflicts with my local changes.

Reply:
Done